src/config/webpack.dev.config.babel.js

```diff
@@ -58,7 +58,7 @@ export default Object.assign({}, webpackConfig, {
       query: BABEL_QUERY,
     }, {
       test: /\.scss$/,
-      loader: 'style!css?modules&importLoaders=2&sourceMap&localIdentName=[local]___[hash:base64:5]!autoprefixer?browsers=last 2 version!sass?outputStyle=expanded&sourceMap',
+      loader: 'style!css?modules&importLoaders=2&sourceMap&localIdentName=[local]!autoprefixer?browsers=last 2 version!sass?outputStyle=expanded',
```

Comment:
I've just noticed sometimes the css isn't loaded on a fresh start-up. I'll look into that...